src/HtmlGenerator/Utilities/DeclaredSymbolInfo.cs

```diff
@@ -68,13 +68,7 @@ public string GetNamespace()
             return description.Substring(0, lastDot);
         }
 
-        public int Weight
-        {
-            get
-            {
-                return MatchLevel * 10 + KindRank;
-            }
-        }
+        public int Weight => (MatchLevel * 10) + KindRank;

```

Comment:
The extra parentheses are redundant and I don't think it's an improvement.

Reply:
It doesn't change the result value due to operator precedence, but it's better to make math stuff more human readable IMO.